src/rewrite.rs

```diff
@@ -37,6 +44,16 @@ impl<'a> RewriteContext<'a> {
             codemap: self.codemap,
             config: self.config,
             block_indent: self.block_indent + self.config.tab_spaces,
+            overflow_indent: self.overflow_indent,
+        }
+    }
+
+    pub fn overflow_context(&self, overflow: usize) -> RewriteContext<'a> {
```

Comment:
Why is there a separation between regular block indentation and overflow indentation? Can't we simply set `block_indent += config.tab_spaces`?